prov/efa/src/rdm/efa_rdm_pke_cmd.c

```diff
@@ -374,16 +374,17 @@ void efa_rdm_pke_handle_data_copied(struct efa_rdm_pke *pkt_entry)
  *      For other types of error, an error EQ entry is written.
  *
  * @param[in]	pkt_entry	pkt entry
- * @param[in]	err		libfabric error code
  * @param[in]	prov_errno	provider specific error code
  */
-void efa_rdm_pke_handle_tx_error(struct efa_rdm_pke *pkt_entry, int err, int prov_errno)
+void efa_rdm_pke_handle_tx_error(struct efa_rdm_pke *pkt_entry, int prov_errno)
 {
 	struct efa_rdm_peer *peer;
 	struct efa_rdm_ope *txe;
 	struct efa_rdm_ope *rxe;
 	struct efa_rdm_ep *ep;
 
+	int err = to_fi_errno(prov_errno);
```

Comment:
Alternatively (or as a future improvement), `err` can be overridden based on some of the conditions below.